packages/field-highlighter/src/vaadin-user-tags.js

```diff
@@ -132,8 +162,35 @@ export class UserTags extends PolymerElement {
   }
 
   /** @private */
-  __targetChanged(target) {
-    this.$.overlay.positionTarget = target;
+  __onTargetVisibilityChange(isVisible) {
+    this.__isTargetVisible = isVisible;
+
+    if (isVisible && this.__flashQueue.length > 0 && !this.flashing) {
```

Comment:
I've looked at this a few times now since the prototype, but I still don't comprehend this condition, mostly because I have no idea how the component works. Maybe we can add a small comment on each case that is handled in this method?

Reply:
Done.